packages/component/src/Activity/StackedLayout.js

```diff
@@ -152,7 +152,7 @@ const StackedLayout = ({
       })}
       role="group"
     >
-      <div className="webchat__stacked-layout__main">
+      <div className="webchat__stacked-layout__main" role="list">
```

Comment:
"list" implies it's a container with multiple items to be narrated.

While it is true that activity could have multiple attachments, the text message itself is not part of attachments. Activity = text message + N x attachment.

So I think "listitem" is not a good role for text message as it is always single.

Reply:
Where does attachment come into this change? An activity has an ordered list of children, one of which is the message. There are various parts to the activity, but the message is the main child of that list.